zebrad/tests/acceptance.rs

```diff
@@ -53,12 +54,21 @@ fn generate_args() -> Result<()> {
     let output = child.wait_with_output()?;
     output.assert_failure()?;
 
+    // create config file path in the same location as binary
+    let binary_path = Path::new(env!("CARGO_BIN_EXE_zebrad"));
+    let path = format!(
+        "{}/{}",
+        binary_path.parent().unwrap().to_str().unwrap(),
+        "zebrad.toml"
+    );
```

Comment:
Suggestion: Use `PathBuf.push`, which will automatically add the correct path separators.

https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.push

Reply:
Thanks, a lot better. Applied at https://github.com/ZcashFoundation/zebra/pull/859/commits/7a9bd4ae38c4bb8766e32bf656e8cf4c65a8e70c